.drone/drone.jsonnet

```diff
@@ -122,7 +155,27 @@ local docker_manifest(app) = {
       for arch in archs
     ],
   },
+] + [
+  // Continuously Deploy to dev env
+  pipeline('cd-to-dev-env') {
+    trigger: {
+      ref: [
+        // always deploy tip of main to dev
+        'refs/heads/main',
+      ],
```

Comment:
This pipeline will trigger immediately when a commit is pushed, together with the Docker image pipeline. Do you expect timing issues here? That is, this pipeline updates the environment and this environment tries to pull an image that does not exist yet.

Reply:
Good point, I'll add a dependency on the previous step (deploy and publish images)